SparkPost/support-docs
# WE-309 backfill content last updated dates

## What Changed
- add lastUpdated to front matter on .md files
- Ran a script locally to backfill these dates based on git history

## How to verify
- Check a few .md files and make sure the lastUpdated date was added and matches the existing support docs / momentum docs site
